EXT_BeamPhysics.md

```diff
@@ -0,0 +1,344 @@
+BeamPhysics Extension to the openPMD Standard for Describing Particle Beams, Photons, and External Fields
+=========================================================================================================
+
+Overview
+--------
+
+The `BeamPhysics` extension to the openPMD standard is meant for describing particles and fields commonly encountered in accelerator physics simulations.
+
+How to Use this Extension
```

Comment:
We potentially do not need this paragraph on technical usage ("how specify the attribute") since the base standard explains this already.

Instead, we can write an introduction here and the requirement to co-use the extension with `SpeciesType`.

Reply:
@ax3l I'm not exactly sure what you are thinking about here. Could you write a proposed change?

Reply:
No problem, can you squash all commits first please?